docs/resources/azapi_operation.md

```diff
@@ -0,0 +1,133 @@
+---
+subcategory: ""
+layout: "azapi"
+page_title: "Generic Azure Resource Operation: azapi_operation"
+description: |-
+  Perform resource function of an existing azure resource
+---
+
+# azapi_operation
+
+This resource can manage any Azure resource manager resource operation.
```

Comment:
Can we emphasize when to use the resource and when to use the data source? From my understanding, the operation that is only used for reading should use the data source.

Also, we shall also add more words in the data source document.